pkg/pfcpsim/session/far_builder.go

```diff
@@ -0,0 +1,88 @@
+package session
+
+import (
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type farBuilder struct {
+	farID       uint32
+	applyAction uint8
+	method      IEMethod
+}
+
+type downlinkFARBuilder struct {
```

Comment:
single builder instance is enough IMO (see my comment below about marking uplink/downlink)